WorkflowCombine/Testing/PublisherTesting.swift

```diff
@@ -43,6 +44,19 @@ extension RenderTester {
             assertions: { _ in }
         )
     }
+
+    @available(*, deprecated, renamed: "expect(publisher:producingOutput:key:)")
```

Comment:
there are a handful of uses in the monorepo that can be migrated to avoid this warning